examples/ord-min-max.js

```diff
@@ -0,0 +1,42 @@
+const {Max, Min, Ord} = require('../fantasy-monoids');
```

Comment:
... or maybe I will just add some unit tests and do the move to mocha later. Currently all of the fl repos use nodeunit. I think when we decide to move to mocha ore something else, we should cleanup all repos.